modules/apps/frontend-js/frontend-js-components-sample-web/src/main/java/com/liferay/frontend/js/components/sample/web/internal/display/context/TranslationManagerDisplayContext.java

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.frontend.js.components.sample.web.internal.display.context;
+
+import com.liferay.portal.kernel.json.JSONArray;
+import com.liferay.portal.kernel.json.JSONFactoryUtil;
+import com.liferay.portal.kernel.json.JSONUtil;
+import com.liferay.portal.kernel.language.LanguageUtil;
+import com.liferay.portal.kernel.util.HashMapBuilder;
+import com.liferay.portal.kernel.util.LocaleUtil;
+import com.liferay.portal.kernel.util.StringUtil;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * @author Carlos Lancha
+ */
+public class TranslationManagerDisplayContext {
+
+	public List<String> getActiveLanguageIds() {
+		if (_activeLanguageIds != null) {
+			return _activeLanguageIds;
+		}
+
+		_activeLanguageIds = new ArrayList<>();
+
+		_activeLanguageIds.add(getDefaultLanguageId());
+		_activeLanguageIds.add("ca_ES");
+		_activeLanguageIds.add("fr_FR");
+
+		return _activeLanguageIds;
+	}
+
+	public Set<Locale> getAvailableLocales() {
+		if (_availableLocales != null) {
+			return _availableLocales;
+		}
+
+		_availableLocales = LanguageUtil.getAvailableLocales();
+
+		return _availableLocales;
+	}
+
+	public JSONArray getAvailableLocalesJSONArray() {
+		if (_availableLocalesJSONArray != null) {
+			return _availableLocalesJSONArray;
+		}
+
+		Set<Locale> availableLocales = getAvailableLocales();
+
+		JSONArray availableLocalesJSONArray = JSONFactoryUtil.createJSONArray();
+
+		for (Locale availableLocale : availableLocales) {
+			String id =
```

Comment:
you can do `LocaleUtil.toW3cLanguageId(locale)` for the "en-US" version and `LocaleUtil.toBCP47LanguageId(locale)` for the "en_US" version